### ISSUE TITLE
Deep dive into the Rasa training policies

### ISSUE THREAD

**Issue by JustinaPetr** [RasaHQ/contributors#4]
Write a tutorial explaining each Rasa training policy and how it works under the hood.

**Comment by rasatester** [comment]
i would like to work on this one

**Comment by arbazkhan971** [comment]
I am interested to write blog on this topic

**Comment by EmmaWightman** [comment]
Hi @arbazkhan971 it's great to hear you're interested in writing a blog post on this topic! We will assign this issue to you, do you have any open questions that you need help with? 

Thanks for the support. :)

**Comment by prakashr7d** [comment]
I like to write a blog on it...
@EmmaWightman

**Comment by EmmaWightman** [comment]
sounds great @prakashr7d , how is it going? Let me know if you have any further questions :)

**Comment by rashmimetri** [comment]
@EmmaWightman Is this still open, i would like to write

**Comment by EmmaWightman** [comment]
Yes this is still open @rashmimetri :) Thank you for your support, let me know if you need anything

**Comment by vishnupriyavr** [comment]
Is this still open?
